task/task-pangolin-ropsten/src/service/redeem/handler.rs

```diff
@@ -102,10 +103,10 @@ impl RedeemHandler {
                 tx.block_number,
                 last_confirmed,
             );
-            tokio::time::sleep(std::time::Duration::from_secs(30)).await;
-            self.sender_to_redeem
-                .send(ToRedeemMessage::EthereumTransaction(tx))
-                .await?;
+            // tokio::time::sleep(std::time::Duration::from_secs(30)).await;
+            // self.sender_to_redeem
+            //     .send(ToRedeemMessage::EthereumTransaction(tx))
```

Comment:
It looks like that `ToRedeemMessage::EthereumTransaction` is useless anymore, and can be removed?